### PR TITLE
Completed css-four-column-layout

### PR DESCRIPTION
**Pull request by Eugenepark215**
Screen Shot 2022-08-29 at 8 33 37 PM

### CODE REVIEW

File: css-four-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  max-width: 2500px;
+  background-color: rgb(247 248 250);
+}
+
+.row {
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  font-family: Barlow, sans-serif;
+}
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:13]: `row` class rulesets should only have `display: flex` applied to them, anything else should be on a separate class that you can add to the element in addition to the `row` class.

File: css-four-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
+.container {
+  box-sizing: border-box;
+  margin: 0 auto;
+  max-width: 2500px;
+  background-color: rgb(247 248 250);
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:5]: Only `max-width` and `margin` on `container` classes, shift the other styling to a separate class that you can add in addition to the `container` class.

File: css-four-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  text-align: center;
+  justify-content: center;
+  font-family: Barlow, sans-serif;
+}
+
+.column-one-fourth {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:19]: `column` classes should only have `flex-basis` or `width` applied to them, shift all other styling to a separate class. I would recommend reviewing `css-layout-classes` for a refresher on the concept and its usage, you seem to have some misunderstandings on the topic currently.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Eugenepark215** [comment]
Screen Shot 2022-08-30 at 12 27 37 PM

File: css-four-column-layout/styles.css [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+.column-full {
+  margin: 0 auto;
+  text-align: center;
+  justify-content: center;
+  font-family: Barlow, sans-serif;
+  background-color: rgb(247 248 250);
+  flex-basis: 100%;
+}
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:17]: `column-` classes should only have `flex-basis` or `width`, this still doesn't address my prior feedback.

File: css-four-column-layout/styles.css [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+.column-one-fourth {
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  flex-basis: calc(100%/4);
+  background-color: rgb(247 248 250);
+}
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:26]: `column-` classes should only have `flex-basis` or `width`, this still doesn't address my prior feedback.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Pull request merged**